siriuspy/siriuspy/injctrl/csdev.py

```diff
@@ -289,3 +297,192 @@ def get_injctrl_propty_database():
     }
     dbase = _csdev.add_pvslist_cte(dbase)
     return dbase
+
+
+def get_biasfb_database():
+    """."""
+    dbase = {
+        'LoopState-Sel': {
+            'type': 'enum', 'value': _ct.OffOn.Off, 'enums': _et.OFF_ON},
+        'LoopState-Sts': {
+            'type': 'enum', 'value': _ct.TopUpSts.Off, 'enums': _et.TOPUPSTS},
+        'MinVoltage-SP': {
+            'type': 'float', 'value': -52, 'unit': 'V',
+            'prec': 1, 'lolim': -120, 'low': -120, 'lolo': -120,
+            'hilim': -30.0, 'high': -30.0, 'hihi': -30.0},
+        'MinVoltage-RB': {
+            'type': 'float', 'value': -52, 'unit': 'V',
+            'prec': 1, 'lolim': -120, 'low': -120, 'lolo': -120,
+            'hilim': -30.0, 'high': -30.0, 'hihi': -30.0},
+        'MaxVoltage-SP': {
+            'type': 'float', 'value': -52, 'unit': 'V',
+            'prec': 1, 'lolim': -120, 'low': -120, 'lolo': -120,
+            'hilim': -30.0, 'high': -30.0, 'hihi': -30.0},
+        'MaxVoltage-RB': {
+            'type': 'float', 'value': -52, 'unit': 'V',
+            'prec': 1, 'lolim': -120, 'low': -120, 'lolo': -120,
+            'hilim': -30.0, 'high': -30.0, 'hihi': -30.0},
+
+        'ModelType-Sel': {
```

Comment:
we should add units for similar PVs (`ModelMaxNumPts-SP/RB`, for example) in order for archiver viewer to be used more easily.

Reply:
I don't understand @xresende. I have added units for this PV you commented. I just didn't add for `-Sel/-Sts` PVs. Do you think I should add for them too?

Reply:
I dont understand either. I meant `ModelMaxNumPts-SP/RB`, as I wrote. But right now I can see on this github.com PR interface that the units are there! I remember looking at the pair `ModelMaxNumPts-SP/RB` and not finding unit in neither of them. And you did not commit any changes... Well, most likely I overlooked something... sorry.

As to `Sel/Sts` PVs, I think we should also add units. Specially if the property is such that other devices might have identical or similar properties. Besides, adding units  you avoid archiver viewer having one axis for `Sel`and another one for `Sts`. I would suggest we copy the name of the property (`ModelType`, for example) to the PV unit field.

Reply:
ok! I can do that!

Reply:
@xresende, I've done that. I'll explain how I did it in the main thread so it is more visible to others.